### ISSUE TITLE
Accessibility violation in the empty canvas page

### ISSUE THREAD

**Issue by JeenaJose123** [elyra-ai/canvas#2342]
Ref: 3971

![Image](https://github.com/user-attachments/assets/4e11e276-455a-44e0-84b4-3b0cd7f49b1b)

**Comment by tomlyn** [comment]
This is fixed. The fix will be in the next Elyra Canvas release.
